spec/support/components/projects/project_life_cycles/edit_dialog.rb

```diff
@@ -50,7 +50,8 @@ def within_async_content(close_after_yield: false, &)
         end
 
         def clear_date_for(step)
-          find("input[id^='project_available_phases_attributes_#{step.position - 1}']").set ""
+          find("input[id^='project_phase_date_range']").set ""
+          find_by_id("edit-project-life-cycles-dialog-title").click
```

Comment:
Why is this needed?

Reply:
To dismiss the datepicker so that the form save button can be clicked.